template.env

```diff
@@ -1,8 +1,8 @@
 DATABASE_NAME=
 DATABASE_USER=
 DATABASE_PASSWORD=
-DATABASE_HOST=
-DATABASE_PORT=
+DATABASE_HOST=mysql
```

Comment:
This is the template file, the default settings are set in installer.sh line 116-